<launch>
    <node pkg="task2" type="client" name="client" output="screen">
        <param name="filename" type="string" value="/home/anifan/rmua/video_source/01.jpg"/>
        <param name="load" type="string" value="hello,world!"/>
        <param name="is_echo" type="bool" value="false"/>
        <param name="loop_times" type="int" value="1000"/>
        <param name="is_with_img" type="bool" value="true"/>
    </node>
</launch>